packages/typescript/src/lib/TypescriptCompiler.ts

```diff
@@ -66,6 +72,20 @@ export class TypescriptCompiler {
     throw new Error('something went wrong generating declaration files');
   }
 
+  private getCompilerProgram(programOptions: ts.CreateProgramOptions) {
+    const { compiler = 'tsc' } = this.options;
```

Comment:
Since normalise options sets all the default values to the options, we don't have to set the default here again.

```suggestion
    const { compiler } = this.options;
```